src/components/Modals/Send/TxFeeRadioGroup.tsx

```diff
@@ -111,7 +111,7 @@ export const TxFeeRadioGroup = ({ fees }: TxFeeRadioGroupProps) => {
             <Amount.Crypto
               fontSize='sm'
               fontWeight='normal'
-              maximumFractionDigits={4}
+              maximumFractionDigits={Math.min(6, asset.precision)}
```

Comment:
I'm concerned about this logic being replicated in so many places and making a mistake in the future.

I think we should look at the Amount.Crypto component and see if we can do something there, like accepting an "assetPrecision" value and doing the Math.min there, if both exist?

Reply:
ok

Reply:
Amount.Crypto is used in quite a few different places, I dont think we have easy access to asset precision in all of those places to pass it in.

Would we just default to use maximumFractionDigits in the case assetPrecision is undefined (not passed in)?

Im not sure that I agree this is better. I made the changes locally (havent pushed yet) and it does dry it up a little bit but at the expensive of making Amount.Crypto more complex and confusing to understand why it works that way

Reply:
Just using a value of `6` should be fine. If the asset.precision is less than 6, the `Amount.Crypto` will show the `value` with the fewer digits. It won't pad the fraction with more 0's to fit the max length of 6.

i.e., `value = '12.3'` results in displaying `12.3` not `12.300000`.
`value=12.34567891` results in displaying `12.345679`

Reply:
changed it back to 6